app/services/nsm/assessment_syncer.rb

```diff
@@ -92,6 +95,13 @@ def sync_further_info_requests
       end
     end
 
+    def sync_youth_court_fee_adjustment
+      return if data['include_youth_court_fee_original'].blank? || !FeatureFlags.youth_court_fee.enabled?
+
+      claim.allowed_youth_court_fee = data['include_youth_court_fee']
```

Comment:
should that not be `allowed_youth_court_fee`?

Reply:
as Patrick mentioned the other day - the way we are doing things is clunky and probably could have been done better but if we want to change the convention it means a hell of a lot of data migration to format the data in the right way (or a "V2" schema with all the bells and whistles that needs to work)